onnxruntime/core/providers/nnapi/nnapi_builtin/builders/model_builder.cc

```diff
@@ -6,6 +6,7 @@
 #include "core/common/logging/logging.h"
 #include "core/common/safeint.h"
 #include "core/common/status.h"
+#include "core/framework/execution_provider.h"
```

Comment:
the additional include seems to be the only change in this file, is it needed?

Reply:
Yes - due to move of DataLayout enum